src/common/routes.js

```diff
@@ -3,13 +3,15 @@ import { Route } from 'react-router';
 import App from './containers/app.js';
 import Builds from './containers/builds.js';
 import BuildDetails from './containers/build-details.js';
-import Home from './containers/home.js';
+import Dashboard from './containers/dashboard.js';
+import Landing from './containers/landing.js';
 import LoginFailed from './containers/login-failed.js';
 import RepositorySetup from './containers/repository-setup.js';
 
 export default (
   <Route component={App}>
-    <Route path="/" component={Home}/>
+    <Route path="/" component={Landing}/>
+    <Route path="/repos" component={Dashboard}/>
```

Comment:
Why not `/dashboard`? If that's what we're using to describe this interface, why shouldn't the user?